src/runtime/markdown-parser/handler/link.ts

```diff
@@ -25,3 +26,11 @@ export default function link (h: H, node: Node) {
 
   return h(node, 'a', props, all(h, node))
 }
+
+function normalizeLink (link: string) {
+  if (isRelative(link)) {
```

Comment:
`isRelative` only detects links starting with `./`, `../`. It will be good if we support links without these prefixes.

```
[test](1.test.md)
```